package.json

```diff
@@ -129,7 +129,7 @@
         "postpack": "shx rm -f oclif.manifest.json oclif.lock",
         "prepack": "yarn clean-all && npm run sf-prepack",
         "sf-prepack": "sf-prepack",
-        "test": "c8 mocha 'test/unit/**/*.test.ts'",
+        "test": "c8 mocha 'test/unit/**/*.test.ts' && c8 report --reporter=lcov",
```

Comment:
Previously `c8` was just generating report locally. By adding a step to `report` using `lcov` as a reporter it creates `lcov.info` file which gets uploaded to codecov along with related json files.